File: app/src/main/java/br/edu/farol/gadoplus/model/TipoGasto.java
package br.edu.farol.gadoplus.model;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "tipo_gasto")
public class TipoGasto{
    @PrimaryKey(autoGenerate = true)
    private int id;

    @ColumnInfo(name = "nome")
    private String nome;

    public TipoGasto(String nome){
        this.nome = nome;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String toString(){ return nome; }
}

File: app/src/main/java/br/edu/farol/gadoplus/model/TipoGasto.kt
package br.edu.farol.gadoplus.model

import androidx.room.PrimaryKey
import androidx.room.ColumnInfo
import androidx.room.Entity

@Entity(tableName = "tipo_gasto")
class TipoGasto(@field:ColumnInfo(name = "nome") var nome: String) {
    @PrimaryKey(autoGenerate = true)
    var id = 0

    override fun toString(): String {
        return nome
    }
}